lib/service/state.go

```diff
@@ -57,9 +57,10 @@ func init() {
 
 // processState tracks the state of the Teleport process.
 type processState struct {
-	process *TeleportProcess
-	mu      sync.Mutex
-	states  map[string]*componentState
+	process        *TeleportProcess
+	mu             sync.Mutex
+	states         map[string]*componentState
+	componentCount int
```

Comment:
Can we emphasise in variable name what is the difference between current component calculated based on `len(states`
and desired `stateOK` `componentCount` ?

Reply:
Tried to improve this in 4c0fca2669aa462cbb800dda71d5fbc71e4c381d.